vendor/k8s.io/apimachinery/pkg/apimachinery/registered/registered.go

```diff
@@ -242,7 +242,12 @@ func (m *APIRegistrationManager) RESTMapper(versionPatterns ...schema.GroupVersi
 	for enabledVersion := range m.enabledVersions {
 		if !unionedGroups.Has(enabledVersion.Group) {
 			unionedGroups.Insert(enabledVersion.Group)
-			groupMeta := m.groupMetaMap[enabledVersion.Group]
+			groupMeta, found := m.groupMetaMap[enabledVersion.Group]
```

Comment:
It's possible that this patch wasn't applied properly during the last merge.  It shouldn't really matter now that TPRs are deprecated.